src/server/utils/onModuleLoad.js

```diff
@@ -147,32 +133,6 @@ export default function onModuleLoad({
     );
   }
 
-  if (requiredExternals) {
```

Comment:
Did all this logic get moved to holocron?

Reply:
yes